### DISCUSSION TITLE
GUI for using pytubefix in pyQT

### DISCUSSION THREAD

**Discussion by fedetony** [JuanBindez/pytubefix#234] [Show and tell]
you can see a Simple GUI here: [ GUI pytubefix ](https://github.com/fedetony/yt_download_pytubefix/tree/master). 
Please support me with comments, bugs or enhancements you would like to include.
![image](https://github.com/user-attachments/assets/d8a10c9f-c7d6-4f37-8fba-2f1a0d8b026e)
Enjoy!

**Comment by BBC-Esq** [comment]
I've created a nice GUI as well located at this repository:  https://github.com/BBC-Esq/pytubefix-GUI
